MapboxCoreNavigation/RouteController.swift

```diff
@@ -942,6 +948,12 @@ extension RouteController: CLLocationManagerDelegate {
         } else {
             routeProgress.currentLegProgress.stepIndex += 1
         }
+        
+        if isDeadReckoningEnabled, let coordinates = routeProgress.currentLegProgress.currentStep.coordinates, let intersections = routeProgress.currentLegProgress.currentStep.intersections {
```

Comment:
If `intersectionDistances` is only set when dead reckoning is enabled, then the documentation for that property should say so.

Reply:
Sure. As soon as we (@bsudekum, @1ec5, myself) gain clarity on whether we are wrapping this inside a `isDeadReckoningEnabled` flag,  I will make a final update on the `intersectionDistances` property accordingly.

Reply:
This code should not be influenced by dead reckoning.

Reply:
Already removed - post our discussion on slack 😺